### DISCUSSION TITLE
pb with skiasharp dll 2.80.3

### DISCUSSION THREAD

**Discussion by guinness57** [QuestPDF/QuestPDF#194] [Q&A]
I try to use questPDF in my project, no probleme in VS2019 the code is well running with my unitTests, but when deplying on iis I got an error :

Could not load file or assembly 'SkiaSharp' or one of its dependencies. Strong name signature could not be verified

tried to si with skiasharp and it seems it's a known issue with this version.

but as it is the version used in QuestPDF , it ssems I can't use it.

Does anyone have a workaround ???

Regards

**Comment by MarcinZiabek** [comment]
Hi 😁 Please try to install explicitly the `SkiaSharp` nuget with forced 2.80.2 version.

**Reply by guinness57** [reply]
thank you for your quick response, unfortunately, when I downgrade to 2.80.2, VS automatically downgrade QuestPDF to v2021.9.3 (but I don't want this version...) 
I mean if the QuestPDF nuget is compiled explicitely with version 2.80.3, how can it refers to lower version ?

**Comment by MarcinZiabek** [comment]
Do you know the latest version of SkiaSharp that does not have this problem? Is there any easy way to validate?

P.S. I am very sorry for the delayed reply. A lot is going on in my personal life: highly positive yet time-consuming events. I hope that you understand 😁

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
